src/solver/misc/options.cpp

```diff
@@ -82,45 +79,53 @@ std::unique_ptr<Yuni::GetOpt::Parser> CreateParser(Settings& settings, StudyLoad
                 "Deprecated, use linear-solver instead.");
 
     //--linear-solver-parameters
-    parser->add(
-      options.solverOptions.linearSolverParameters,
-      ' ',
-      "linear-solver-parameters",
-      "Set linear solver-specific parameters, for instance --linear-solver-parameters=\"THREADS 1 "
-      "PRESOLVE 1\""
-      "for XPRESS or --linear-solver-parameters=\"parallel/maxnthreads 1, lp/presolving TRUE\" for "
-      "SCIP."
-      "Syntax is solver-dependent, and only supported for SCIP & XPRESS.");
+    parser->add(options.solverOptions.linearSolverParameters,
+                ' ',
+                "linear-solver-parameters",
+                "Linear solver-specific parameters, for instance \"THREADS 1 "
+                "PRESOLVE 1\""
+                " for XPRESS or \"parallel/maxnthreads 1, lp/presolving TRUE\" for "
+                "SCIP. Syntax is solver-dependent, and only supported for SCIP & XPRESS.");
 
     //--solver-parameters
     parser->add(options.solverOptions.linearSolverParameters,
                 ' ',
                 "solver-parameters",
                 "Deprecated, use linear-solver-parameters instead.");
 
+    // --lp-solver-param-optim-1
+    parser->add(options.solverOptions.lpSolverParamOptim1,
+                ' ',
+                "lp-solver-param-optim-1",
+                "Linear solver-specific parameters for first optimization."
+                " Only supported for SCIP & XPRESS.");
+
+    // --lp-solver-param-optim-2
+    parser->add(options.solverOptions.lpSolverParamOptim2,
+                ' ',
+                "lp-solver-param-optim-2",
```

Comment:
```suggestion
                "linear-solver-parameters-optim2",
```

Reply:
We finally opted for **--lp-solver-param**, **--lp-solver-param-optim-1** and **--lp-solver-param-optim-2**

Please see this[ overall PR ](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2765)